webapp/src/components/formComponents/JDKVersionSelectForm.tsx

```diff
@@ -21,6 +21,7 @@ const ProductSelectForm: React.FC<Props> = props => {
 
         const onJDKVersionChange = (value: string) => {
             product.jdk = value
+            product.packageName = jdkVersion!.packageNames.length >= 0 ? jdkVersion!.packageNames[0] : ""
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Unexpected_type

Reply:
It should be packageNames.length > 0, not >= ... my bad, could you please fix it? :)

Reply:
Well, sure, but this patch is generally not working. Not product, not jdk verson, not anything do exists here in time of call. You may see the desperate ! in the line. What is the variable to use?

Reply:
fixed, the >; however the PR still do not work.I humbly ask for your assistance on this my first react code.